charts/prometheus-elasticsearch-exporter/templates/podmonitor.yaml

```diff
@@ -1,7 +1,8 @@
 {{- if and .Values.podMonitor.enabled .Values.serviceMonitor.enabled }}
 {{- fail "Either .Values.podMonitor.enabled or .Values.serviceMonitor.enabled can be enabled at a time, but not both." }}
 {{- else if .Values.podMonitor.enabled }}
-apiVersion: monitoring.coreos.com/v1
+{{- if .Capabilities.APIVersions.Has .Values.podMonitor.apiVersion}}
```

Comment:
This conditional should not be present, a literal is just being replaced with a variable after all.

Reply:
Thanks @zeritti. I have removed the check from pod and svc monitors. Please take a look. Also, how does the release process work? If there a scheduled release for the charts in this repo?